### ISSUE TITLE
Question on 10 missing keys

### ISSUE THREAD

**Issue by mathis1337** [Coding-Enthusiast/FinderOuter#41]
have a question. Recently there was a flood and a paper notebook containing a offline wallet was damage and it destroyed part of a WIF private key, so now I basically have:

Kw**********(I have the next 40 characters, just not posting for obvious reasons), so I am missing 10 characters in all.

I also have the public key. Is it possible to use this software to start a search at Kw... and iterate over the missing 10 characters with the known 40 characters also in the key.

For example : 1B2Q8vPm5E5b8yxaNWUW5ZfdCR5Zu1KMJn - is the public address

and for the WIF private key I'd have Kw**********jcQmKRPNTF8CU9H1thzC981DCrZgmS4m8ygXFCEk (this is just an example, no funds are here)

basically I want to always have search for private key start at 'Kw', then search for missing 10, and end with 'jcQmKRPNTF8CU9H1thzC981DCrZgmS4m8ygXFCEk'

Any advice is appreciated and examples are even more appreciated. 

Or is 10 characters just too many possible combos. All advice is much appreciated! I can maybe take a guess at one more character that kind of looks like something and get it down to 9 missing characters, but I am still worried that is too many.

Thank you
